otterdog/eclipse-kura.jsonnet

```
@@ -153,13 +153,13 @@ orgs.newOrg('iot.kura', 'eclipse-kura') {
       allow_rebase_merge: false,
       allow_squash_merge: true,
       code_scanning_default_setup_enabled: false,
-      default_branch: "master",
+      default_branch: "develop",
       description: "Applications for Eclipse Kura™ framework",
       delete_branch_on_merge: false,
```

Let's improve our QOL a little


```suggestion
      delete_branch_on_merge: true,
```